### DISCUSSION TITLE
Questions in entities

### DISCUSSION THREAD

**Discussion by ahmm16** [drodil/backstage-plugin-qeta#247] [Q&A]
Hi @drodil I have the following question of one member of my team: 

"" Hi,
Right after posting questions, I’m redirected to the common “Questions” page, then I need to navigate to the entity back again.

Question is not assigned to the entity automatically. So, if you forgot to select current entity a question won’t be visible in “Q&A” tab for an entity. Is this behavior? ""

**Comment by ahmm16** [comment]
Taking advantage of this section, why can't I create more than one question for an entity now?

![image](https://github.com/user-attachments/assets/8cc8354f-b1ed-4973-b330-3f410a1410fd)

In previous versions I could

![image](https://github.com/user-attachments/assets/5d4a6670-a591-4888-bec2-d1bbae071011)

**Comment by drodil** [comment]
I will take a look at this at some point. There must have been something I missed when reverted back to MUI4

**Reply by ahmm16** [reply]
My fault man! I forgot the property showAskButton. I thougth it was a predefined prop with true. But I have another question. Let me explain in a new comment.

**Comment by ahmm16** [comment]
Sorry again for the previous questions. I was mixing up concepts and got confused.

I confused the first question because this "Go ahead and ask one" button does not fill in the entity even if you are on that entity's page. I was reviewing the code and it seems to be because it does not pass the entity as filters. I understand how it works and it seems correct to me, but don't you think it could be a little confusing if I am on the entity's page tab?
![image](https://github.com/user-attachments/assets/4342ba83-7771-4237-8214-b37aae58766f)

![image](https://github.com/user-attachments/assets/b09c6701-2fe2-4c52-8245-39cf0a013798)

**Reply by drodil** [reply]
Ok, I will take a look!

**Reply by drodil** [reply]
Published a new version 3.15.0 which has this fixed and other improvements!

**Reply by ahmm16** [reply]
Thank you as always!! Tomorrow, I will test it for sure!


Thxx!!

**Reply by drodil** [reply]
Please take at least 3.15.1 as there was a bug with tag moderation in 3.15.0 :) Also 3.15.2 available with support for entity/tag pickers to use virtualization - especially useful with large catalogs!
